cmd/govim/gopls_client.go

```diff
@@ -20,30 +20,47 @@ var _ protocol.Client = (*govimplugin)(nil)
 
 func (g *govimplugin) ShowMessage(ctxt context.Context, params *protocol.ShowMessageParams) error {
 	g.logGoplsClientf("Asked to show a message: %v", params.Message)
+	if params.Type == protocol.Error {
+		opts := make(map[string]interface{})
+		opts["mousemoved"] = "any"
+		opts["moved"] = "any"
+		opts["padding"] = []int{0, 1, 0, 1}
+		opts["wrap"] = false
+		opts["close"] = "click"
+
+		g.ChannelCall("popup_create", params.Message, opts)
```

Comment:
We need to schedule the showing of a popup like this:

https://github.com/govim/govim/blob/40a414920272565dd9dcda9de5009a492858e66d/cmd/govim/gopls_client.go#L90

I also need to write up some docs/wiki explaining the threading model in `govim` and why this is necessary, but that's beyond this review.